api/src/main/java/org/hawkular/apm/api/services/Criteria.java

```diff
@@ -48,7 +50,6 @@
     private String hostName;
     private long upperBound;
     private long lowerBound;
-    private String principal;
```

Comment:
Just wondering if it isn't better to keep pincipal as a property here. As this class is used to query data from clients.